addons/sale_timesheet/views/project_portal_templates.xml

```diff
@@ -12,10 +12,11 @@
             <th t-if="display_sol">Sales Order Item</th>
         </th>
         <xpath expr="//tr/td[t[@t-esc='timesheet.name']]" position="after">
-            <td  t-if="display_sol">
-	        <t t-if="timesheet.so_line.order_id.access_url and so_accessible"><a t-att-href="'%s' % timesheet.so_line.order_id.access_url"><t t-out="timesheet.so_line.display_name"/></a></t>
-		<t t-else=""><t t-out="timesheet.so_line.display_name"/></t>
-	    </td>
+            <td t-if="display_sol">
+                <t t-if="timesheet.so_line.order_id.access_url and so_accessible"><a t-att-href="timesheet.so_line.order_id.access_url"><t t-out="timesheet.so_line.display_name"/></a></t>
+                <t t-elif="timesheet.so_line.display_name"><t t-out="timesheet.so_line.display_name"/></t>
+                <t t-else=""><span class="text-muted">Non-billable</span></t>
+            </td>
```

Comment:
```suggestion
            <td t-if="display_sol">
                <t t-if="timesheet.so_line.order_id.access_url and so_accessible"><a t-att-href="timesheet.so_line.order_id.access_url"><t t-out="timesheet.so_line.display_name"/></a></t>
                <t t-elif="timesheet.so_line.display_name t-out="timesheet.so_line.display_name"/></t>
                <span class="text-muted"  t-else="">Non-billable</span>
            </td>
```
Lines 15 and 16 have formatting changes. It’s fine if they don’t affect the commit history.